ngx_http_upstream_check_module.c

```diff
@@ -241,6 +242,7 @@ struct ngx_http_upstream_check_srv_conf_s {
     ngx_array_t                             *fastcgi_params;
 
     ngx_uint_t                               default_down;
+    ngx_uint_t                               ssl;
```

Comment:
Indeed for having both `default_down` and `ssl`, but I would skip this change to avoid changing unnecessary things. IMHO, it makes it hard to sync with upstream.